subprojects/core/src/integTest/groovy/org/gradle/api/file/ProjectLayoutIntegrationTest.groovy

```diff
@@ -217,7 +217,7 @@ class ProjectLayoutIntegrationTest extends AbstractIntegrationSpec {
         'FileCollection'             | 'Closure'        | 'project.layout.filesFor({ "%s/src/resource/file.txt" })'
         'FileCollection'             | 'List'           | 'project.layout.filesFor([ "%s/src/resource/file.txt" ])'
         'FileCollection'             | 'array'          | 'project.layout.filesFor([ "%s/src/resource/file.txt" ] as Object[])'
-        'FileCollection'             | 'FileCollection' | 'project.layout.filesFor(new org.gradle.api.internal.file.collections.SimpleFileCollection(new File("%s/src/resource/file.txt")))'
+        'FileCollection'             | 'FileCollection' | 'project.layout.filesFor(org.gradle.api.internal.file.collections.ImmutableFileCollection.of(new File("%s/src/resource/file.txt")))'
```

Comment:
Nit: you could replace `org.gradle.api.internal.file.collections.ImmutableFileCollection` with `${ImmutableFileCollection.name}` to keep this shorter and refactor-safe.

Reply:
good idea.  done.